common/lib/xmodule/xmodule/modulestore/split_mongo/split.py

```diff
@@ -1059,7 +1059,7 @@ def get_item(self, usage_key, depth=0, **kwargs):
                 log.debug("Found more than one item for '{}'".format(usage_key))
             return items[0]
 
-    def get_items(self, course_locator, settings=None, content=None, qualifiers=None, **kwargs):
+    def get_items(self, course_locator, settings=None, content=None, qualifiers=None, in_tree_only=False, **kwargs):
```

Comment:
@doctoryes @adampalay or we can set it to `discard_orphans` name as well ? what do you say. That way it would be more clear and would default to False.